kubernetes/kind/README.md

```diff
@@ -5,12 +5,12 @@
 ### Usage
 
 1. Update the `hostPath` property in all `configs/**/kind.yaml` files to match the path of the project on your filesystem
-1. (optional) Update the `hostPort` property in all `configs/**/kind.yaml` files to the port that suits you the best to access portainer instance. Default ports are, by context:
-  * base: 9010
+1. (optional) Update the `hostPort` property in all `configs/**/kind.yaml` files to the port that suits you the best to access portainer instance. Default ports (http, https) are, by context:
+  * base: 9010, 9011
   * ingress: 9020
   * multi-master: 9030
   * taints: 9040
-  * agent: 9050
+  * agent: 9050, 9051
```

Comment:
I suggest using xxx3 port for TLS as xxx0 is by convention used for http and xxx1 for agent.
9000 (portainer default port) -> xxx0
9001 (portainer agent port) -> xxx1
9443 (portainer https port) -> xxx3

Reply:
you make alot of sense to me :)